[2023-09-27 19:31:59,201] INFO 172.30.0.1 - - [27/Sep/2023 19:31:59] "POST /rota_backbone HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
[2023-09-27 19:32:54,941] INFO Adicionando Rota de Backbone de nome: 'ROTA RIO DE JANEIRO - BELO HORIZONTE' - call_trace=/home/python/app/app.py L45  
[2023-09-27 19:32:54,959] INFO Adicionando Rota de Backbone: RotaBackbone(id=2, nome='ROTA RIO DE JANEIRO - BELO HORIZONTE', uf_ponta_a='RJ', cidade_ponta_a='RIO DE JANEIRO', uf_ponta_b='MG', cidade_ponta_b='BELO HORIZONTE') - call_trace=/home/python/app/app.py L53  
[2023-09-27 19:32:54,960] INFO 172.30.0.1 - - [27/Sep/2023 19:32:54] "POST /rota_backbone HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
[2023-09-27 19:33:03,563] INFO Coletando Rotas de Backbone - call_trace=/home/python/app/app.py L75  
[2023-09-27 19:33:03,564] INFO 2 Rotas de Backbone encontradas - call_trace=/home/python/app/app.py L85  
[2023-09-27 19:33:03,565] INFO 172.30.0.1 - - [27/Sep/2023 19:33:03] "GET /rotas_backbone HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
[2023-09-27 19:35:42,978] INFO Deletendo dados sobre a Rota de Backbone #1 - call_trace=/home/python/app/app.py L128 
[2023-09-27 19:35:43,004] INFO Deletendo Rota de Backbone #1 - call_trace=/home/python/app/app.py L137 
[2023-09-27 19:35:43,004] INFO 172.30.0.1 - - [27/Sep/2023 19:35:43] "DELETE /rota_backbone?id=1 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
[2023-09-27 19:35:51,012] INFO Deletendo dados sobre a Rota de Backbone #2 - call_trace=/home/python/app/app.py L128 
[2023-09-27 19:35:51,035] INFO Deletendo Rota de Backbone #2 - call_trace=/home/python/app/app.py L137 
[2023-09-27 19:35:51,035] INFO 172.30.0.1 - - [27/Sep/2023 19:35:51] "DELETE /rota_backbone?id=2 HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
[2023-09-27 19:36:09,287] INFO Coletando Rotas de Backbone - call_trace=/home/python/app/app.py L75  
[2023-09-27 19:36:09,288] INFO 172.30.0.1 - - [27/Sep/2023 19:36:09] "GET /rotas_backbone HTTP/1.1" 200 - - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
[2023-09-27 19:36:35,182] INFO  * Detected change in '/home/python/app/app.py', reloading - call_trace=/usr/local/lib/python3.9/site-packages/werkzeug/_internal.py L187 
